internal/proto/controller/api/services/v1/auth_method_service.proto

```diff
@@ -96,17 +96,17 @@ service AuthMethodService {
 }
 
 message GetAuthMethodRequest {
-  string id = 1; // @gotags: `class:"public"`
+  string id = 1; // @gotags: `class:"public" eventstream:"observation"`
 }
 
 message GetAuthMethodResponse {
   resources.authmethods.v1.AuthMethod item = 1;
 }
 
 message ListAuthMethodsRequest {
-  string scope_id = 1 [json_name = "scope_id"]; // @gotags: `class:"public"`
-  bool recursive = 20 [json_name = "recursive"]; // @gotags: `class:"public"`
-  string filter = 30 [json_name = "filter"]; // @gotags: `class:"public"`
+  string scope_id = 1 [json_name = "scope_id"]; // @gotags: `class:"public" eventstream:"observation"`
+  bool recursive = 20 [json_name = "recursive"]; // @gotags: `class:"public" eventstream:"observation"`
+  string filter = 30 [json_name = "filter"]; // @gotags: `class:"public" eventstream:"observation"`
```

Comment:
I'm not sure about this one, given that it could contain anything the user types.

Reply:
I think, we can include `recursive` if we want, since it's not something users type in.... but whether or not we need it: I'll leave that for you and PM to decide.